bot/code_review_bot/tasks/clang_tidy_external.py

```diff
@@ -42,10 +42,12 @@
 
 CLANG_MACRO_DETECTION = re.compile(r"^expanded from macro")
 
+BUILD_HELP_MSG = """For private static analysis, please see [our private docs in Mana](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=130909687), if you cannot access this resource, ask your reviewer to help you resolve the issue."""
+
 
 class ExternalTidyIssue(ClangTidyIssue):
     """
-    An issue reported by source-test-clang-tidy-external
+    An issue reported by source-test-clang-external
```

Comment:
Do we really want this change? For me this is sounds like an external compiler set based on clang and not clang-tidy based static-analysis.